packages/react-native-session-replay/android/src/main/kotlin/com/datadog/reactnative/sessionreplay/ColorUtils.kt

```diff
@@ -0,0 +1,53 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.reactnative.sessionreplay
+
+import androidx.annotation.VisibleForTesting
+import com.facebook.react.views.view.ReactViewGroup
+
+private const val ALPHA_CODE_LENGTH: Int = 2
+private const val HEX_RADIX: Int = 16
+private const val OPAQUE_ALPHA_VALUE: Int = 255
+private const val HEX_COLOR_INCLUDING_ALPHA_LENGTH: Int = 8
+
+internal fun resolveAlpha(opacity: Float, backgroundColor: Int?): Int {
```

Comment:
I'm a bit confused by this method. It means, that if `backgroundColor` has alpha bits set, we completely ignore `opacity`, it is kind of non-obvious behavior. Why this method takes both parameters and not take only `backgroundColor`, extracts alpha and then compares with `opacity` somewhere else?